src/forms/NewDocumentForm.js

```diff
@@ -4,6 +4,27 @@ import {Button, GridContainer} from 'kf-uikit';
 import SelectElement from '../components/FileAnnotation/SelectElement';
 import {fileTypeDetail} from '../common/fileUtils';
 
+const plachelderText = {
+  SHM: {
+    title: 'Biospecimen shipping manifest from ...',
+    desc:
+      'Shipping manifest for biospecimens shipped from LabA to CenterB on 00/00/0000 .....',
+  },
+  CLN: {
+    title: `20XX Phenotypic data manifest for  ...`,
+    desc:
+      'Phenotypic data for 00 patients and 00 containing demographics, diagnoses, and ...',
+  },
+  SEQ: {
+    title: 'Sequencing Center manifest from [Seq. Center Name] on [date]... ',
+    desc: 'Manifest received from [Seq. Center Name] accounting for ...',
+  },
+  OTH: {
+    title: '...data sheet...',
```

Comment:
Is there other wording perhaps a little more helpful?  Maybe `Clinician Observation Form`